website/homepage/upload_demos.py

```diff
@@ -67,10 +64,11 @@ def upload_demo_to_space(
         )
```

Comment:
Nit: you don't have to rename `run.py` to `app.py` and could instead change `app_file` to `run.py`

Reply:
thanks @abidlabs, I set the app_file to run.py now. I also updated all the spaces to remove the app.py file and reuploaded with run.py